plugins/plugin-chart-table/src/TableChart.tsx

```diff
@@ -366,37 +380,17 @@ export default function TableChart<D extends DataRecord = DataRecord>(
     updateExternalFormData(setDataMask, pageNumber, pageSize);
   };
 
-  const totalsFormatted =
-    totals &&
-    columnsMeta
-      .filter(column => Object.keys(totals).includes(column.key))
-      .reduce(
-        (acc: { value: string; className: string }[], column) => [
-          ...acc,
-          {
-            value: formatColumnValue(column, totals[column.key])[1],
-            className: column.dataType === GenericDataType.NUMERIC ? 'dt-metric' : '',
-          },
-        ],
-        [],
-      );
-
-  const totalsHeaderSpan =
-    totalsFormatted &&
-    columnsMeta.filter(column => !column.isPercentMetric).length - totalsFormatted.length;
-
   return (
     <Styles>
       <DataTable<D>
         columns={columns}
-        totals={totalsFormatted}
-        totalsHeaderSpan={totalsHeaderSpan}
         data={data}
         rowCount={rowCount}
         tableClassName="table table-striped table-condensed"
         pageSize={pageSize}
         serverPaginationData={serverPaginationData}
         pageSizeOptions={pageSizeOptions}
+        width={width}
```

Comment:
This was accidentally deleted here: 

https://github.com/apache-superset/superset-ui/commit/2af6a2441c75f3556fbf42a9e2d1ceae9e0579a0#diff-1b5ecf1086dbd4b0e942970dc4e4b20077a2d30d6046d08444b46d8cb57de2a4L303